LanguageFeatures/Wildcards/binding_A01_t13.dart

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion A local declaration whose name is `_` does not bind that name to
+/// anything.
+///
+/// @description Checks that it is a compile-time error to use a variable named
+/// `_` in a string literal.
```

Comment:
Perhaps?:
```suggestion
/// @description Checks that an identifier reference of the form `_` in a string literal does
/// not resolve to a local variable which is a wildcard since there is no such binding.
```

Reply:
Updated